integration_tests/mockData/actionPlanByPrisonNumberData.ts

```diff
@@ -43,11 +43,30 @@ const actionPlans = {
               },
             ],
             createdBy: 'auser_gen',
+            createdByDisplayName: 'George Costanza',
             createdAt: '2023-07-20T09:29:15.386Z',
+            createdAtPrison: 'BXI',
             updatedBy: 'auser_gen',
+            updatedByDisplayName: 'George Costanza',
             updatedAt: '2023-07-20T09:29:15.386Z',
+            updatedAtPrison: 'BXI',
             targetCompletionDate: '2124-01-29',
             notes: 'Billy will struggle to concentrate for long periods.',
+            goalNotes: [
```

Comment:
Updated the wiremock stub (that is used by the cypress tests) for returning a prisoner's action plan/goals to include the new `goalNotes` array , plus a couple of other fields that were missing (added them for completeness)